packages/atomic/src/components/commerce/atomic-commerce-recommendation-interface/atomic-commerce-recommendation-interface.tsx

```diff
@@ -206,7 +206,6 @@ export class AtomicCommerceRecommendationInterface
       i18n: this.i18n,
       store: this.store,
       interfaceElement: this.host,
-      // TODO: KIT-4333: remove this and use injectStylesForNoShadowDOM decorator instead
```

Comment:
More of a note to myself, but I should do this here https://github.com/coveo/ui-kit/pull/5483/files#diff-49f05522d8a6a741f08096adc577eb763fb43656f2572791c63055b2d1d6b4efR303 once this PR is merged.